### PR TITLE
Allowing token files in CLI

### PR DESCRIPTION
**Pull request by Hunman**
Allow a filename as command line argument where we can fetch the token from

### CODE REVIEW

File: cheat.php [hunk 1]
```diff
@@ -58,10 +66,14 @@
 
 if( strlen( $Token ) !== 32 )
 {
-	Msg( 'Failed to find your token. Verify token.txt' );
+	Msg( 'Failed to find your token. Verify ' . $TokenFile );
```
Comment (Zarthus) [inline cheat.php]: Sounds like this can be NULL.

File: cheat.php [hunk 2]
```diff
@@ -58,10 +64,14 @@
...
 	exit( 1 );
 }
 
+if (isset($TokenFile)) {
```
Comment (Zarthus) [inline cheat.php]: If it's always set, the if is unnecessary.

Comment (Hunman) [reply cheat.php]: And now the if is off

**Comment by xPaw** [comment]
Why do people keep on institing on providing a file? Are CLI and ENV variables not enough already?

**Comment by Hunman** [comment]
It simplifies using multiple accounts much more than CLI or ENV variables, especially if you have to stop them to update it frequently

**Comment by xPaw** [comment]
What stops you making a file that starts the script with said params? You end up with same amount of files.

**Comment by Hunman** [comment]
Technically nothing

**Comment by SeinopSys** [comment]
`php cheat.php $(cat token-accountname.txt)` woks flawlessly for me on Linux, I don't really see a need for a file argument.

**Comment by xPaw** [comment]
Yep.

**Pull request closed without merging**